modules/appeals_api/app/services/appeals_api/events/appeal_received.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module Events
+    class AppealReceived
+      def initialize(opts)
+        @opts = opts
+        raise InvalidKeys unless required_keys?
+      end
+
+      def hlr_received
+        return unless Flipper.enabled?(:decision_review_hlr_email)
+
+        log_error(guid, 'HLR') unless email
+
+        template_type = 'higher_level_review_received'
```

Comment:
@cilestin this is the tentative name of the template, but we can just adjust it later